sdk/core/azure_core_amqp/.dict.txt

```diff
@@ -1,5 +1,7 @@
 mgmt
 sasl
+amqp
```

Comment:
IMO, you could put "amqp" in the main dictionary (.cspell.json file). We'll probably end up referring to it at least as existing enough. Not a big deal either way.

Reply:
AMQP *is* in the main dictionary. But it's not picked up for reasons which I don't fully understand.

Reply:
In what file extension(s) is the word being flagged? Maybe I can try down why.

Note: this isn't blocking. Just trying to reduce maintenance hassles.

Reply:
azure-sdk-for-rust/sdk/eventhubs/azure_messaging_eventhubs/src/error.rs.

And oh, this is *lovely*. I saved cspell.json (with no modifications) and the errors disappeared. So this may be a VS code bug of some kind.